test/form-layout.html

```diff
@@ -545,6 +545,24 @@
         expect(estimateEffectiveColspan(layout.children[0])).to.be.closeTo(2, .1);
       });
 
+      it('should update style after layout-item is not hidden', async() => {
+        sinon.spy(layout, '_invokeUpdateStyles');
+        const firstDisplay = layout.firstElementChild.style.display;
+        layout.firstElementChild.setAttribute('hidden', true);
+        container.shadowRoot.querySelector('vaadin-form-layout').firstElementChild.style.display = 'none';
+        layout.firstElementChild.removeAttribute('style');
+        let isStyleAfterRemovedNullOrEmpty = false;
+        if (layout.firstElementChild.getAttribute('style') === ''
+            || layout.firstElementChild.getAttribute('style') === null) {
+          isStyleAfterRemovedNullOrEmpty = true;
+        }
+        expect(isStyleAfterRemovedNullOrEmpty).to.be.true;
+        layout.firstElementChild.removeAttribute('hidden');
+        container.shadowRoot.querySelector('vaadin-form-layout').firstElementChild.style.display = firstDisplay;
```

Comment:
Why query for this again? `container.shadowRoot.querySelector('vaadin-form-layout')` is the same as `layout`

Reply:
yes, it is the same element. I changed it